src/main/java/com/deloitte/elrr/services/dto/ExtensibleDto.java

```diff
@@ -0,0 +1,18 @@
+package com.deloitte.elrr.services.dto;
+
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Data
+@EqualsAndHashCode(callSuper = true)
+public abstract class ExtensibleDto extends AuditableDto {
+
+    private Map<String, Object> extensions;
```

Comment:
we should enforce IRIs for keys.

Reply:
https://github.com/US-ELRR/elrrservices/pull/73/commits/7fad15f318d762abb75ccc30f70117df6efc1bab